src/aui-form-validator/tests/unit/js/tests.js

```diff
@@ -602,6 +602,160 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(input.hasClass('success-field'));
         },
 
+        /*
+         * @tests AUI-2820
+         */
+        'should add success-field and aria-invalid attr on all, named Nodes and their fieldContainer': function() {
```

Comment:
This 2 tests are failing on my machine when run with gulp test, could you run `gulp test`  check this?

✖ [aui-form-validator]: Passed: 17 Failed: 2 Total: 19 (ignored 0) (0.254 seconds)
    should add success-field class to, and remove aria-invalid attr from all, named Nodes and their fieldContainer
       All fields should have success-field
       Expected: true (boolean)
       Actual: false (boolean)
    should add success-field class to, and remove error-field class and aria-invalid attr from all, named Nodes and their fieldContainer
       All fields should have success-field
       Expected: true (boolean)
       Actual: false (boolean)